selenium_webdriver/src/test/java/tests/sauceDemoTests/SauceDemoCartBadgePresenceTest.java

```diff
@@ -0,0 +1,24 @@
+package tests.sauceDemoTests;
+
+import model.User;
+import org.testng.annotations.Test;
+import pageobject.sauceDemoPages.SauceDemoLoginPage;
+import service.UserCreator;
+import tests.BaseTest;
+
+import static org.testng.Assert.assertTrue;
+
+public class SauceDemoCartBadgePresenceTest extends BaseTest {
+
+    @Test
+    public void cartBadgeWasAddedToTheCartLogoAfterAddingItemIntoCart() {
+        User testUser = UserCreator.withCredentialsFromProperty();
+        String badgeWasAddedToTheCartLogo = String.valueOf(new SauceDemoLoginPage(driver)
+                .openSauceDemoLoginPage()
+                .loginToSauceDemoMainPage(testUser)
+                .clickAddToCartButton()
+                .cartBadgeIsPresentAfterItemWasAddedToItCheck());
+
+        assertTrue(true, "Badge was not successfully added to the cart logo.");
```

Comment:
incorrect assertion. TRUE always be TRUE. Please look into assertTrue()

Reply:
Fixed